samples/notebooks/M-Admin/extra_user_apps/apps/codeserver_proxy/scripts/getcreds.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
```

Comment:
where is this file called from ?

Reply:
It is a helper function to get credentials onto the instance to support AWS Toolkit.  It is not invoked automatically by any build process

Reply:
This is now invoked at bootstrap of the image